patchwork-events-entity/src/main/java/com/patchworkmc/mixin/event/entity/MixinPlayerEntity.java

```diff
@@ -96,6 +96,17 @@ private float hookApplyDamageForHurtEvent(float damage, DamageSource source) {
 	private void hookApplyDamageForHurtEventCancel(DamageSource source, float damage, CallbackInfo info) {
 		if (damage <= 0) {
 			info.cancel();
-		}
-	}
+        }
+    }
+    
+    @Inject(method = "handleFallDamage", at = @At(value = "INVOKE", target = "net/minecraft/entity/LivingEntity.handleFallDamage(FF)V", shift = At.Shift.AFTER), cancellable = true)
```

Comment:
I think it would be better to shadow `abilities` and just check it in the other mixin instead